roadmap.md

```diff
@@ -45,6 +45,10 @@ See [`contributing.md`](./contributing.md)
 
 - Drop `react`
 
+### Track
+
+- Track `novnc` versions, when a new version is released we should update it.
```

Comment:
Can you please add information about the version number we are waiting for?
If you have the information about the estimated time of the `novnc` release it would be helpful as well.

Reply:
thanks @sharvit , done